sdk/framework/backend.go

```diff
@@ -486,9 +486,15 @@ func (b *Backend) handleRootHelp() (*logical.Response, error) {
 		return nil, err
 	}
 
+	// The plugin type (e.g. "kv", "cubbyhole") is only assigned at the time
+	// the plugin is enabled (mounted). If specified in the request, the type
```

Comment:
Can you explain a bit more why you'd want to specify `requestResponsePrefix`?

Reply:
The plugins/backends currently don't have a notion of their type. It defaults to the name of the executable but can be overridden when the plugin is mounted. Since we need this type to form the Request & Response full names, I'm passing them as an optional request parameter to the plugin root help endpoint.

Reply:
Should I add more explanation to the comment?

Reply:
Yes I think an addition to the comment would be good.